packages/liveblocks/src/room.ts

```diff
@@ -1003,6 +1003,12 @@ See v0.13 release notes for more information.
     if (state.connection.state === "connecting") {
       updateConnection({ ...state.connection, state: "open" });
       state.numberOfRetry = 0;
+
+      // Re-broadcast the user presence during a reconnect.
+      if (state.lastConnectionId !== undefined) {
+        updatePresence(state.me);
```

Comment:
I think it's slightly better if we do:

```suggestion
        state.buffer.presence = state.me;
        tryFlushing();
```

That way, presence subscribers will not be notified since the presence didn't change.